features/step_definitions/web_steps.rb

```diff
@@ -106,7 +106,7 @@ def with_scope(locator)
   end
 end
 
-When /^(?:|I )choose "([^"]*)"(?: within "([^"]*)")?$/ do |field, selector|
```

Comment:
I think we need to remove the selector argument too.

```
Your block takes 2 arguments, but the Regexp matched 1 argument. (Cucumber::ArityMismatchError)
features/step_definitions/web_steps.rb:109:in `/^(?:|I )choose "(.*)"$/'
/home/travis/.rvm/rubies/ruby-2.3.4/bin/bundle:30:in `block in <main>'
/home/travis/.rvm/rubies/ruby-2.3.4/bin/bundle:22:in `<main>'
features/admins/users/admin_abuse_users.feature:102:in `And I choose "Lift permanent suspension, effective immediately."'
```

Reply:
Fuck me in the eye. Will do!

Reply:
No, we need your eye(s). Should be good to merge when tests pass.